p5js/00_ImageClassification_MultipleImages/sketch.js

```diff
@@ -0,0 +1,82 @@
+/* ===
+ML5 Example
+00_ImageNet_Simple
+Simple Image Classification using p5.js
+=== */
+
+// Initialize the ImageNet method with the MobileNet model.
+const classifier = new ml5.ImageClassifier('MobileNet');
+
+let img;
+currentIndex = 0;
```

Comment:
Sorry, noticed one more thing! These should probably all have a `let` and the two `[]` lines should have a semi-colon. This is so great though -- I'll merge and we can do a little cleanup after if need be!

Reply:
Thank you so much! I'll start using that format going forward.